.github/workflows/acceptance-public.yml

```
@@ -0,0 +1,112 @@
+name: Acceptance Tests on Public Network
+
+on:
+  workflow_dispatch:
+    inputs:
+      operator_id:
+        description: 'The Account ID of the operator'
+        required: true
+        type: string
+      operator_key:
+        description: 'The ED25519 DER encoded private key of the operator'
```

nit: key can be either type
```suggestion
        description: 'The ED25519 or ECDSA DER encoded private key of the operator'
```